monai/auto3dseg/utils.py

```diff
@@ -372,3 +373,149 @@ def algo_from_pickle(pkl_filename: str, template_path: PathLike | None = None, *
         algo_meta_data.update({k: v})
 
     return algo, algo_meta_data
+
+
+def list_to_python_fire_arg_str(args: list) -> str:
+    """
+    Convert a list of arguments to a string that can be used in python-fire.
+
+    Args:
+        args: the list of arguments.
+
+    Returns:
+        the string that can be used in python-fire.
+    """
+    args_str = ",".join([str(arg) for arg in args])
+    return f"'{args_str}'"
+
+
+def check_and_set_optional_args(params: dict) -> str:
+    """ """
+    cmd_mod_opt = ""
+    for k, v in params.items():
+        if isinstance(v, dict):
+            raise ValueError("Nested dict is not supported.")
+        elif isinstance(v, list):
+            v = list_to_python_fire_arg_str(v)
+        cmd_mod_opt += f" --{k} {str(v)}"
+    return cmd_mod_opt
+
+
+def _create_default(cmd: str, cmd_prefix: str = "python", **kwargs: Any) -> str:
```

Comment:
I think this function name to too generic..

Reply:
Thanks! Here are a few candidates and I am also open to suggestions:
- `_create_cmd_default`
- `_create_auto3dseg_default`
- `_prepare_cmd_default`
- `_prepare_cmd_python`

Reply:
thanks, I think `_prepare_cmd_default` is good here